### PR TITLE
parser: Add unit test for range translations

### PR DESCRIPTION
**Pull request by hlyytine**
Signed-off-by: Joonas Onatsu <joonasx@ssrc.tii.ae>
Signed-off-by: Hannu Lyytinen <hannux@ssrc.tii.ae>

### CODE REVIEW

**Comment by hlyytine** [comment]
Hi @kent-mcleod , this PR is for adding license to test_rpi4.dts as you requested in #108 . So in the Python file we already had the license info (we just added copyright mention for our mods), the device tree source on the other hand is autogenerated from running guest with `dtc -I fs /proc/device-tree`. The original test.dts has Linux as author and GPL-2.0 as a license. So I tried to mimick that, only to find out CI here says it's not permissive enough. Could you have a look at this problem? To me it makes sense to block GPL but why test.dts passes?

File: camkes/parser/tests/test_rpi4.dts [hunk 1]
```diff
@@ -0,0 +1,376 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0-only
+ * Copyright Linux
+ */
```
Comment (kent-mcleod) [inline camkes/parser/tests/test_rpi4.dts:4]: For test_rpi4.dtb I believe the tool just requires a new file `test_rpi4.dtb.license` with the contents the same as the 4 lines of this file.

**Comment by kent-mcleod** [comment]
> Could you have a look at this problem? To me it makes sense to block GPL but why test.dts passes?

The CI appears to just have an issue with the binary file: https://github.com/seL4/camkes-tool/pull/116#discussion_r955685418

**Comment by kent-mcleod** [comment]
That seems to have fixed it. Thanks!

**Review by kent-mcleod — APPROVED** [review]

**Pull request merged**